packages/react-charting/src/utilities/UtilityUnitTests.test.ts

```diff
@@ -1,5 +1,7 @@
 import * as utils from './utilities';
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29548&commitId=4c7e98524016488c0146990e869dfbb205024960&env=prod&clientType=FLUENTUI)


  <details><summary><b>Signals</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Signals.Blocked.default.chromium.png|676| Changed|
|Signals.Emailed.default.chromium.png|667| Changed|
|Signals.Comments.default.chromium.png|644| Changed|
|Signals.Comments (count).default.chromium.png|704| Changed|
|Signals.Awaiting approval.default.chromium.png|997| Changed|
|Signals.New (positioning).default.chromium.png|12| Changed|
|Signals.Malware detected.default.chromium.png|1019| Changed|
|Signals.Missing metadata.default.chromium.png|986| Changed|
|Signals.Unseen reply.default.chromium.png|820| Changed|
|Signals.Mention.default.chromium.png|685| Changed|
|Signals.Warning.default.chromium.png|710| Changed|
|Signals.Unseen edit.default.chromium.png|781| Changed|
|Signals.New.default.chromium.png|12| Changed|
|Signals.Someone checked out.default.chromium.png|1105| Changed|
|Signals.Trending.default.chromium.png|698| Changed|
|Signals.You checked out.default.chromium.png|957| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>